chain/consensus/mir/config.go

```diff
@@ -0,0 +1,69 @@
+package mir
+
+import (
+	"github.com/filecoin-project/go-address"
+
+	"github.com/filecoin-project/mir/pkg/checkpoint"
+
+	"github.com/filecoin-project/lotus/chain/ipcagent/rpc"
+)
+
+// ---
+
+type BaseConfig struct {
+	ID            address.Address
+	DatastorePath string
+	// InitialCheckpoint from which to start the validator.
+	InitialCheckpoint *checkpoint.StableCheckpoint
+	// CheckpointRepo determines the path where Mir checkpoints
+	// will be (optionally) persisted.
+	CheckpointRepo string
+	// The name of the group of validators.
+	GroupName string
```

Comment:
How is this value used?

Reply:
It is used for debugging. In tests the groupname is test name